extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -520,6 +522,18 @@ public void onValueReceived(String variable, String value, String service) {
                     updateState(LINEIN, newState);
                     break;
                 }
+                case "TOSLinkConnected": {
+                    State newState = UnDefType.UNDEF;
+                    if (stateMap.get("TOSLinkConnected") != null) {
+                        if (stateMap.get("TOSLinkConnected").equals("true")) {
```

Comment:
Nothing that needs to be changed, but this could be shorten using:
```java
if (Objects.equals("true", stateMap.get("TOSLinkConnected"))) {
    newState = OnOffType.ON;
} else {
    newState = OnOffType.OFF;
}